components/tooltip/test/tooltip.test.js

```diff
@@ -4,13 +4,20 @@ import { runConstructor } from '../../../tools/constructor-test-helper.js';
 
 const basicFixture = html`
 	<div>
-		<div id="implicit-target" tabindex="-1">
+		<div id="implicit-target" tabindex="-1" role="button">
```

Comment:
What's the purpose of the `role` on the implicit target?

Reply:
It was added without it because it generated warning logs in the test output because the tooltip was used in a non-accessible way 😅